### DISCUSSION TITLE
extend to end

### DISCUSSION THREAD

**Discussion by reosari** [keiwando/toonsquid#372] [Ideas]
i dont know if this is already a feature but i dont want to drag 1 drawing across 600 frames when i could be animating instead (edit also maybe an extend to certain frame number)

**Comment by Cat-Butler** [comment]
Set your "current frame marker/selector" in the timeline to any keyframe e.g. in your example, frame 600.
Select the layer you want to extend, and it will extend automatically from the last frame on that layer to the currently selected frame.

https://github.com/keiwando/toonsquid/assets/25494772/0fd64bb5-03a1-4990-805b-1fca2dd9eca2
